### PR TITLE
APIDOC-253: created documentation breadcrumbs

### PR DESCRIPTION
**Pull request by lggg123**
## Description

Added proper ability to jump to page by creating breadcrumbs below the subtitle for the documentation landing page.

### CODE REVIEW

File: lib/nexmo_developer/.rubocop.yml [hunk 1]
```diff
@@ -216,7 +216,7 @@ Style/GlobalStdStream:
   Enabled: true
 
 Style/OptionalBooleanParameter:
-  Enabled: true
+  Enabled: false
```
Comment (marcoranieri) [inline lib/nexmo_developer/.rubocop.yml:219]: Please revert it

File: lib/nexmo_developer/app/helpers/static_helper.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
+module StaticHelper
```
Comment (marcoranieri) [inline lib/nexmo_developer/app/helpers/static_helper.rb:1]: We don't want to add another layer of abstraction here - would you mind remove this helper file please?

File: lib/nexmo_developer/app/presenters/sidenav.rb [hunk 3]
```diff
@@ -82,6 +82,17 @@ def items
     end
   end
 
+  def map_items(nav_items, index, index_nil = false)
```
Comment (marcoranieri) [inline lib/nexmo_developer/app/presenters/sidenav.rb:85]: This is not part of this PR - please could you remove/revert it?

File: lib/nexmo_developer/app/views/static/landing_page_documentation.html.erb [hunk 4]
```diff
@@ -2,14 +2,26 @@
   <div class="center spacious content-block">
     <h1><%= @landing_info['title'].html_safe %></h1>
     <p><%= raw @landing_info['subtitle'].html_safe %></p>
-  </div>
+    <p>
+      <!-- Tthi is the buttons group area--> 
+      <strong>Jump to: </strong>
+      <% @landing_info['top-section'].each do |product| %>
+        <a href="#<%=refactor_keys(product)%>" class="<%= refactor_class %>"><%= first(product) %></a>
```
Comment (marcoranieri) [inline lib/nexmo_developer/app/views/static/landing_page_documentation.html.erb:9]: Let's do a first step by using what we have in plain air and don't worry if it looks to verbose - we'll have time to improve/refactor this bits of code later on

File: lib/nexmo_developer/app/views/static/landing_page_documentation.html.erb [hunk 5]
```diff
@@ -2,14 +2,26 @@
...
+      <%end %>
 
-  <div>
+      <% @landing_info['bottom-section'].each do |product| %>
+        <% product[product.keys.first].each do |cards| %>
+          <a href="#<%=refactor_keys(cards)%>" class="<%= refactor_class %>"><%= first(cards) %></a>
```
Comment (marcoranieri) [inline lib/nexmo_developer/app/views/static/landing_page_documentation.html.erb:14]: same

File: lib/nexmo_developer/config/code_languages.yml [hunk 6]
```diff
@@ -0,0 +1 @@
+/Users/georgelugo/Desktop/vonage/nexmo-developer/config/code_languages.yml
```
Comment (marcoranieri) [inline lib/nexmo_developer/config/code_languages.yml:1]: Please remove this as well

File: lib/nexmo_developer/app/views/static/landing_page_documentation.html.erb [hunk 7]
```diff
@@ -2,14 +2,26 @@
...
+      <%end%>
+    </p>
+  </div>
+<div>
 
 <!-- TOP section -->
     <% @landing_info['top-section'].each do |product| %>
       <hr class="hr--tall Vlt-gradient--blue-to-pink"></hr>
-      <h2><%= product.keys.first %></h2>
+      <h2 id="<%= product.keys.first.to_s.gsub(' ', '-') %>"><%= product.keys.first %></h2>
```
Comment (marcoranieri) [inline lib/nexmo_developer/app/views/static/landing_page_documentation.html.erb:24]: What do you think about using `product.keys.first.parameterize` to create IDs instead of manually doing `product.keys.first.to_s.gsub(' ', '-')`?

File: lib/nexmo_developer/app/views/static/landing_page_documentation/partials/_big_product_card.html.erb [hunk 8]
```diff
@@ -5,7 +5,7 @@
       <svg class="Vlt-icon Vlt-icon--larger Vlt-purple"><use xlink:href="/symbol/volta-icons.svg#<%= products['icon'] %>" /></svg>
     </div>
 
-    <h3 class="Vlt-purple pb-20">
+    <h3 class="Vlt-purple pb-20" id="<%= products.keys.first.to_s.gsub(' ','-') %>">
```
Comment (marcoranieri) [inline lib/nexmo_developer/app/views/static/landing_page_documentation/partials/_big_product_card.html.erb:8]: Also here `product.keys.first.parameterize`

File: lib/nexmo_developer/app/views/static/landing_page_documentation.html.erb [hunk 9]
```diff
@@ -2,14 +2,26 @@
   <div class="center spacious content-block">
     <h1><%= @landing_info['title'].html_safe %></h1>
     <p><%= raw @landing_info['subtitle'].html_safe %></p>
-  </div>
+    <p>
+      <!-- This is the buttons group area-->
+      <strong>Jump to: </strong>
+      <% @landing_info['top-section'].each do |product| %>
+        <a href="#<%=product.keys.first.to_s.gsub(' ', '-')%>"  class='Vlt-btn Vlt-btn--blue Vlt-btn--app Vlt-btn--small'><%= product.keys.first %></a>
```
Comment (marcoranieri) [inline lib/nexmo_developer/app/views/static/landing_page_documentation.html.erb:9]: What do you think about using `product.keys.first.parameterize` to create IDs instead of manually doing `product.keys.first.to_s.gsub(' ', '-')`?

File: lib/nexmo_developer/app/views/static/landing_page_documentation.html.erb [hunk 10]
```diff
@@ -2,14 +2,26 @@
...
+      <%end %>
 
-  <div>
+      <% @landing_info['bottom-section'].each do |product| %>
+        <% product[product.keys.first].each do |cards| %>
+          <a href="#<%=cards.keys.first.to_s.gsub(' ', '-')%>" class='Vlt-btn Vlt-btn--blue Vlt-btn--app Vlt-btn--small'><%= cards.keys.first %></a>
```
Comment (marcoranieri) [inline lib/nexmo_developer/app/views/static/landing_page_documentation.html.erb:14]: Here too (`product.keys.first.parameterize`)

File: lib/nexmo_developer/app/views/static/landing_page_documentation.html.erb [hunk 11]
```diff
@@ -2,14 +2,26 @@
   <div class="center spacious content-block">
     <h1><%= @landing_info['title'].html_safe %></h1>
     <p><%= raw @landing_info['subtitle'].html_safe %></p>
-  </div>
+    <p>
+      <!-- This is the buttons group area-->
+      <strong>Jump to: </strong>
+      <% @landing_info['top-section'].each do |product| %>
+        <a href="#<%=product.keys.first.parameterize %>"  class='Vlt-btn Vlt-btn--blue Vlt-btn--app Vlt-btn--small'><%= product.keys.first %></a>
```
Comment (marcoranieri) [inline lib/nexmo_developer/app/views/static/landing_page_documentation.html.erb:9]: Would you mind change classes for those buttons please?

```
Vlt-btn Vlt-btn--primary Vlt-btn--small

File: lib/nexmo_developer/app/views/static/landing_page_documentation.html.erb [hunk 12]
```diff
@@ -2,14 +2,26 @@
...
+      <%end %>
 
-  <div>
+      <% @landing_info['bottom-section'].each do |product| %>
+        <% product[product.keys.first].each do |cards| %>
+          <a href="#<%=cards.keys.first.parameterize %>" class='Vlt-btn Vlt-btn--blue Vlt-btn--app Vlt-btn--small'><%= cards.keys.first %></a>
```
Comment (marcoranieri) [inline lib/nexmo_developer/app/views/static/landing_page_documentation.html.erb:14]: same please

```
Vlt-btn Vlt-btn--primary Vlt-btn--small

File: lib/nexmo_developer/app/views/static/landing_page_documentation.html.erb [hunk 13]
```diff
@@ -2,14 +2,26 @@
...
+      <%end%>
+    </p>
+  </div>
+<div>
 
 <!-- TOP section -->
     <% @landing_info['top-section'].each do |product| %>
       <hr class="hr--tall Vlt-gradient--blue-to-pink"></hr>
-      <h2><%= product.keys.first %></h2>
+      <h2 id="<%= product.keys.first.parameterize  %>"><%= product.keys.first %></h2>
```
Comment (marcoranieri) [inline lib/nexmo_developer/app/views/static/landing_page_documentation.html.erb:24]: What do you think about adding the ID property to the `<hr>` tag just above each title so it will have more space when the user click on a 'jump to' button

```
<hr id="<%= product.keys.first.parameterize %>" class="hr--tall Vlt-gradient--blue-to-pink"></hr>

Comment (lggg123) [reply lib/nexmo_developer/app/views/static/landing_page_documentation.html.erb]: changed it and it does give it more space.

File: lib/nexmo_developer/app/views/static/landing_page_documentation.html.erb (hunk 12 above)
Comment (lggg123) [reply lib/nexmo_developer/app/views/static/landing_page_documentation.html.erb]: I thought Nick said to make it tertiary. Ill change it to primary and we can discuss tomorrow.

**Review by marcoranieri — APPROVED** [review]
